docs/index.md

```diff
@@ -1,87 +1,34 @@
 # HTTP Adapter
 
-**This is the documentation for HTTP Adapter and it's software components.**
+**This is the documentation for the Httplug web client abstraction and the other PHP-HTTP components.**
 
-The HTTP Adapter abstracts from PHP HTTP clients that are based on [PSR-7](http://www.php-fig.org/psr/psr-7/).
+Httplug abstracts from HTTP clients written in PHP that are based on [PSR-7](http://www.php-fig.org/psr/psr-7/).
```

Comment:
I don't think it is correct, like we want to provide guzzle 5 adapter where the client is not based on PSR-7.